src/frontend/apps/drive/src/features/explorer/components/tree/ExplorerTree.tsx

```diff
@@ -181,15 +191,18 @@ export const ExplorerTree = () => {
     );
   };
 
+  console.log(treeContext?.treeData);
+
   return (
     <div className="explorer__tree">
       <ExplorerTreeActions
         openCreateFolderModal={createFolderModal.open}
         openCreateWorkspaceModal={createWorkspaceModal.open}
-        openSearchModal={searchModal.open}
       />
       <HorizontalSeparator withPadding={false} />
 
+      <ExplorerTreeMobile />
```

Comment:
Nothing too serious, but it bothers me a little to have to go into the component, then go and find the right css class and then go into the scss file to see when it is not displayed.Maybe just use ui-kit's useResponsive hook? WDYT ?

Reply:
I tend to prefer using CSS as media queries are the web gold standard to show or hide elements. Unless it's a really expensive component to render tbh